projects/golang/go/1.16/rpmbuild/SOURCES/golang-gdbinit

```diff
@@ -0,0 +1 @@
+add-auto-load-safe-path /usr/lib/golang/src/runtime/runtime-gdb.py
```

Comment:
What does this do?

Reply:
gdb is the GNU debugger; Golang includes tests for GDB compatibility. 

In GDB, you can load helper scripts which make it easier to convert your data structures to debugger calls and do other helpful things; in order to include these external helper scripts in the tests, you need to explicitly set their path and allow them to be imported safely.

This gdbinit file does that, and was included with the upstream RPM as a source; we're just carrying it over here